src/fare-contracts/ValidityLine.tsx

```diff
@@ -142,7 +142,7 @@ const useAnimatedVerticalLineOffset = (animate: boolean | undefined = true) => {
         easing: Easing.linear,
       }),
     ).start();
-  }, [animate]);
+  }, [animate, animatedOffset]);
```

Comment:
Just a quick question - why is `animatedOffset` defined as `useRef(new Animated.Value(0)).current` instead of just inputting `new Animated.Value(0)` inline to `Animated.timing`??

Reply:
The first argument to the `Animated.timing` is the output value, and is continuously updated during the animation effect. The value is then used in an actual animated component like, in the `transform` function in an `Animated.View`. If just using inline `new Animated.Value(0)`, we would have no reference to it and not able to use it on animated components.

https://reactnative.dev/docs/animations